assets/scss/pages/auth-page.scss

```diff
@@ -0,0 +1,289 @@
+.c-auth__page {
```

Comment:
@mgbelegu same as for the other pages. Please use the `p-` prefix. Also, I think the class should be named like: `p-auth` since `p-` already implies it's a page ;).